README.md

```diff
@@ -60,61 +60,27 @@ A [Dockerfile](Dockerfile) is provided for convenience.
-UNIQUE_APP_ID | The MongoDB database name to store data in (default: `registration`)
+MONGO_URL | The URL to the MongoDB server (default: `mongodb://localhost/registration`)
 VERSION_HASH | The Git short hash used to identify the current commit (default: parsed automatically from the `.git` folder, if it exists)
-*SOURCE_VERSION* | Same as `VERSION_HASH` but overrides it if present. Used by Deis.
-*WORKFLOW_RELEASE_CREATED_AT* | Provided by Deis (default: `null`)
-*WORKFLOW_RELEASE_SUMMARY* | Provided by Deis (default: `null`)
+ADMIN_KEY_SECRET | A API key used to authenticate as admin an access the GraphQL api (default: random key that changes every server restart)
 COOKIE_MAX_AGE | The `maxAge` of cookies set in milliseconds (default: 6 months) **NOTE: this is different from the session TTL**
 COOKIE_SECURE_ONLY | Whether session cookies should sent exclusively over secure connections (default: `false`)
 PASSWORD_RESET_EXPIRATION | The time that password reset links sent via email should be valid for in milliseconds (default: 1 hour)
 SESSION_SECRET | The secret used to sign and validate session cookies (default: random 32 bytes regenerated on every start up)
-GITHUB_CLIENT_ID | OAuth client ID for GitHub *required*
-GITHUB_CLIENT_SECRET | OAuth client secret for GitHub *required*
-GOOGLE_CLIENT_ID | OAuth client ID for Google *required*
-GOOGLE_CLIENT_SECRET | OAuth client secret for Google *required*
-FACEBOOK_CLIENT_ID | OAuth client ID for Facebook *required*
-FACEBOOK_CLIENT_SECRET | OAuth client secret for Facebook *required*
+GROUND_TRUTH_URL | Base URL of [Ground Truth](https://github.com/HackGT/ground-truth) instance (e.g. `https://login.hack.gt`) *required*
+GROUND_TRUTH_ID | OAuth client ID from Ground Truth *required*
+GROUND_TRUTH_SECRET | OAuth client secret from Ground Truth *required*
 EMAIL_FROM | The `From` header for sent emails (default: `HackGT Team <hello@hackgt.com>`)
 EMAIL_KEY | The SendGrid API key for sending emails (default: *none*)
 ADMIN_EMAILS | A JSON array of the emails of the users that you want promoted to admin status when they create their account (default: none)
 EVENT_NAME | The current event's name which affects rendered templates and sent emails (default: `Untitled Event`)
 STORAGE_ENGINE | The name of the storage engine that handles file uploads as defined in [storage.ts](server/storage.ts) (default: `disk`)
 STORAGE_ENGINE_OPTIONS | JSON-encoded object containing options to be passed to the storage engine. Must at least contain a value for the `uploadDirectory` key. For the default `disk` storage engine, this directory is relative to the app's root, can be absolute, and will be created if it doesn't exist. (default: `{ "uploadDirectory": "uploads" }`)
+DEFAULT_TIMEZONE | Timezone used for dates and times (default: `America/New_York`)
 MAX_TEAM_SIZE | The maximum number of users allowed per team (default: `4`)
 QUESTIONS_FILE | Specify a path for the `questions.json` file.
```

Comment:
for consistency, mark as required or indicate the default (ideally for all env vars that don't have this since it makes it easier for new devs to get off the ground)